packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -48,8 +48,8 @@ export type FetchTypeInternal = FetchType | "cache";
 
 export const Odsp409Error = "Odsp409Error";
 
-// Please update the README file in odsp-driver-definitions if you change the defaultCacheExpiryTimeoutMs.
-export const defaultCacheExpiryTimeoutMs: number = 2 * 24 * 60 * 60 * 1000;
+// Must be less than policy of 5 days
+export const defaultCacheExpiryTimeoutMs: number = 2 * 24 * 60 * 60 * 1000; // 2 days in ms
```

Comment:
Why not change this to 5 days? I think your document calls out why but I can't recall.

Reply:
Less churn in the ground truth of how old of unreferenced object may be revivable.  This way we actually can trust the data we'll get from the old code we aren't patching (v2-int.1.1 and before), where it's hard-coded to 2 days.